### ISSUE TITLE
Build script to bundle all imports and main script in one file.

### ISSUE THREAD

**Issue by geirmarius** [geirmarius/bifrost-i18n#6]
Should build both a `bifrost.js` and a `bifrost.min.js` for production.

**Comment by nallown** [comment]
You can use webpack.js module bundler for this. It's pretty simple to use, check it out https://webpack.js.org/.
Their getting started guide is pretty simple to follow as well https://webpack.github.io/docs/tutorials/getting-started/

**Comment by geirmarius** [comment]
Yeah, have been researching different ways I want to do it. Most likely now is webpack, but I'm still open for other solutions. :)
